src/components/others/CustomAccordion.tsx

```diff
@@ -130,7 +130,7 @@ export function CustomAccordion(props: CustomAccordionProps) {
 							!isOpen && "hidden",
 						)}
 					>
-						<div className="pb-4 pt-0" ref={accordionContentRef}>
+						<div className="pb-4 pt-1" ref={accordionContentRef}>
 							{props.children}
 						</div>
 					</div>
```

Comment:
We Should not change the base style here, I think you only want to change the styles for sidebar accordions and not all accordions

Reply:
where else do we use accordions?

Reply:
Whereever the <Details> component is used